string robot_name
geometry_msgs/Pose goal
int32 priority
float32 robot_size